.env

```diff
@@ -13,38 +13,43 @@
 # Run "composer dump-env prod" to compile .env files for production use (requires symfony/flex >=1.2).
 # https://symfony.com/doc/current/best_practices.html#use-environment-variables-for-infrastructure-configuration
 
+# Proxy conf
+HTTP_PROXY_URL=
+HTTPS_PROXY_URL=
+
 # API Platform distribution
 MERCURE_SUBSCRIBE_URL=https://localhost:1337/.well-known/mercure
 VARNISH_URL=http://cache-proxy
 
-###> symfony/framework-bundle ###
+#> symfony/framework-bundle ###
 APP_ENV=dev
-APP_SECRET=!ChangeMe!
+APP_SECRET=SemH5FxV5y4z35v
 TRUSTED_PROXIES=127.0.0.0/8,10.0.0.0/8,172.16.0.0/12,192.168.0.0/16
 TRUSTED_HOSTS='^localhost|api$'
-###< symfony/framework-bundle ###
+#< symfony/framework-bundle ###
```

Comment:
You're breaking the symfony parser

Reply:
I had to do that because in the Makefile I include the .env file. If I launc the make help command this will be outputed:

![image](https://user-images.githubusercontent.com/23189983/116200413-984b2200-a738-11eb-81f1-60559f502582.png)

I'll rewrite the Makefile regex and patch.

Reply:
Revert + patch the regex #5c0d1c2